README.md

```diff
@@ -15,6 +15,17 @@ An app that fetches latest news and headlines. It also allows the users to favou
 
 Also Room was used for adding offline support.
 
+## MVVM Architecture 
```

Comment:
The heading should be "Project Conventions" and under it the title should be "MVVM".
Also under MVVM add the following line "The project follows the MVVM design pattern"